test/valgrind/valgrindTestSuite.sh

```diff
@@ -590,7 +634,10 @@ then
 
     if [ "$lost" != "0" ]
     then
-      failedTest "test/valgrind/$vtest.*" $vtest $lost "valgrind" $xTestNo
+      leakFound "test/valgrind/$vtest.*" $vtest $lost "valgrind" $xTestNo
```

Comment:
Does the if here mean that the script detec either leak fails or mem error fails? (leak seems to take precedence in the case of errors of both types)